deposit_contract/contracts/validator_registration.v.py

```diff
@@ -6,7 +6,7 @@
 MAX_64_BIT_VALUE: constant(uint256) = 18446744073709551615  # 2**64 - 1
```

Comment:
18446744073709551615 deposit to tdmllc1@outlook.com in USD

Reply:
19590.000947185572952524 deposit in binance tdmllc0@gmail.com 3 12 2019 3:30 central in USD